src/main/java/com/online/store/service/OrderServiceImpl.java

```diff
@@ -0,0 +1,7 @@
+package com.online.store.service;
+
+import com.online.store.lib.Service;
+
+@Service
+public class OrderServiceImpl implements OrderService{
```

Comment:
```suggestion
public class OrderServiceImpl implements OrderService {
```
same for all service implementations

Reply:
done